sharing/index.md

```diff
@@ -0,0 +1,47 @@
+---
+tags:
+  - sharing
+author:
+  display_name: AGRC Office
+  email: agrc@utah.gov
+date: 2017-01-26 23:40:06
+title: Sharing GIS Data
+categories: []
+---
+
+#### Quick Links
+
+[Submitting Data to the SGID]({% link sharing/process/index.md %}) — [Basic Metadata Creation Process]({% link sharing/metadata/index.md %}) — [AGRC Sharing Policies]({% link about/policy/index.md %})
```

Comment:
**thought**: not sure i like this nav style

Reply:
Yeap, like I said, can and will be formatting problems. This was just a rough stab. I tried mirroring the style used on /about, but the paragraph wrapping is weird. I'm open to suggestions—the problem I'm trying to solve is that, with the nested folder structure, there's no way to know what else is in that "section" unless you read the entire body of the page.

Reply:
yup, we can work something out there. my apologies for noting a formatting issue.